search-service/src/main/kotlin/com/egm/stellio/search/model/AttributeInstanceResult.kt

```diff
@@ -8,5 +8,6 @@ data class AttributeInstanceResult(
     val instanceId: URI? = null,
```

Comment:
can it be null?

Reply:
Yes, it happens in case we aggregate values for normal representation.